src/plugins/identity/README.md

```diff
@@ -10,10 +10,132 @@ GitHub and Discourse (potentially with a different username on each service).
 We would like to combine these two identities together, so that we can
 represent that user's combined cred properly. The Identity plugin enables this.
 
-Specifically, the instance maintainer can provide a (locally unique) username
-for the user, along with a list of aliases the user is known by, e.g.
-`github/username` and `discourse/other_username`. The aliases are simple string
-representations, that are intended to be easy to maintain by hand in a
-configuration file. Then, the identity plugin will provide a list of
-`NodeContraction`s that can be used by `Graph.contractNodes` to combine the
-user identities as described.
+## Status and Caveats
+
+The Identity plugin currently only applies to new (Weighted)Graphs and Cred
+scores being generated. When including historical Graphs/Cred scores, for
+example to distribute Grain, you may need to contract the identities again.
```

Comment:
"for example to distribute Grain" I think should be in () instead?

Reply:
That seems more like style than a grammar rule. The example seems important enough to take it from abstract "historical Graphs/Cred scores" to a concrete use-case, that I picked commas over parens :]

Reply:
Yes it's just against my preference, can't argue that it's wrong.